src/main/java/com/thirdcc/webapp/repository/DebtRepository.java

```diff
@@ -11,5 +15,6 @@
 @SuppressWarnings("unused")
 @Repository
 public interface DebtRepository extends JpaRepository<Debt, Long> {
-
+    
+    Page<Debt> findAllByStatusInOrderByEventAttendeeIdAscCreatedDateAsc(Pageable pageable, Set<DebtStatus> debtStatus);
```

Comment:
I think can pass the sorting params through pageable object? If it is performed by this way, does that mean we will always get by this order?

Reply:
I think will need to make a test case to test this xia, because i also not really sure if pageable has order also which one will come out first, but if need to pass the sorting params through pageable object de hua will need to add it in front end

Reply:
Resolved by https://github.com/Five-Fishes/Club-Management-Spring-Server/pull/59/commits/c3c818f0366065911619658628e0fd604ddcfe31 , will pass sorting params through pageable object instead of directly putting the order in the query